src/examples/source/GUI.cpp

```diff
@@ -96,6 +96,20 @@ int main(int argc, char **argv)
   bool show_feature_heatmap_plot = false; // injection vs. feature for a particular metavalue
   bool show_calibrators_line_plot = false; // peak area/height ratio vs. concentration ratio
 
+  // Chromatogram display option
```

Comment:
The GUI options defined at the top of the script are starting to get a bit lengthy.  What are your thoughts on another issues where we look to move these into an e.g., `GUISettings` file?  Or we could add them to the `SessionHandler` object.

Reply:
Yes I was not sure about where to put that. SessionHandler was one place, i feel uncomfortable putting legend display flag and the others in that object. I'm not sure about the responsibility of the SessionHandler. 
But most of these statics added in this PR, would be better to be non static, and put in the ChromatogramPlotWidget. That would be the right place. instead of creating a ChromatogramPlotWidget on the stack when drawing, we would instanciated it when it is opened.
This would open the way to have multiple chromatograms (in the future, who knows ...).
But it would need some work, little out of scope of this task, so I would recommend to do it in another task, after that.
I would also agree to move it in session handler for the moment, if you feel it's better like this.